src/components/FlowPopover.vue

```diff
@@ -63,11 +63,11 @@
 }
 
 .flow-popover__link-name { @apply
-  shrink
   truncate
 }
 
 .flow-popover__link-icon { @apply
+  shrink-0
```

Comment:
think this was backwards. 

i noticed the icon shrinking as i brought the width down. this is to stop that - the icon should stay the same size.